whylabs_toolkit/monitor/builder/README.md

```diff
@@ -0,0 +1,34 @@
+# Monitor Builder workflow
+
+This package gives users a workflow to author and modify existing WhyLabs Monitors, 
+using a Builder-Manager approach.
+
+```python
+builder = MissingDataMonitorBuilder(
+    org_id=os.environ["ORG_ID"],
+    dataset_id=os.environ["DATASET_ID"],
+    monitor_id="my-awesome-monitor-3",
```

Comment:
added dataset_id and org_id to be required from the same `ConfigVars` enum we already have
also added a `MonitorCredentials` dataclass, which will either to have `dataset_id` as an argument or get from the env var.
not comfortable setting everything with env vars for the monitor stuff, since users will most likely iterate through monitor_ids and dataset_ids
see if that works